kuma/core/migrations/0004_create_subscription_banner_flag.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def create_subscription_banner_flag(apps, schema_editor):
+    Flag = apps.get_model("waffle", "Flag")
+    Flag.objects.get_or_create(
+        name="subscription_banner", staff=True, note="Shows the subscription banner"
+    )
```

Comment:
If this created the flag, `Flag.get_all()` will not pick it up and you have to wait 1 hour. 
So what you do is:
```python
flag, created = Flag.objects.get_or_create(
        name="subscription_banner", staff=True, note="Shows the subscription banner"
    )
flag.flush()
```

Reply:
Totally a nit but good practice. 
Also, I think we should have do it something like this:
```
if not Flag.objects.filter(name="subscription_banner").exists():
    flag = Flag.objects.create(
        name="subscription_banner", staff=True, note="Shows the subscription banner"
    )
    flag.flush()
```